libmscore/chord.cpp

```diff
@@ -1241,14 +1282,20 @@ qreal Chord::defaultStemLength()
 
       Spatium progression = score()->styleS(Sid::shortStemProgression);
       qreal shortest      = score()->styleS(Sid::shortestStem).val();
+      if (hookIdx) {
+            if (up()) {
+                  if (shortest < 3.0)
+                        shortest = 3.0;
+                  }
+            else {
+                  if (shortest < 3.5)
+                        shortest = 3.5;
+                  }
+            }
 
       qreal normalStemLen = small() ? 2.5 : 3.5;
-      switch (hookIdx) {
-            case 3: normalStemLen += small() ? .5  : 0.75; break; //32nd notes
```

Comment:
We lost the comments describing magic numbers. Could you please add more info in the comments to `hookAdjustment()` method to describe magic numbers and calculations in general?